spec/rpc/README.md

```diff
@@ -0,0 +1,1267 @@
+# RPC spec
+
+This file defines the JSON-RPC spec of Tendermint. This is meant to be implemented by all clients.
+
+## Support
+
+  |              | Tendermint-Go | Tendermint-Rs |
+  |--------------|:-------------:|:-------------:|
+  | JSON-RPC 2.0 |       ✅       |       ✅       |
+  | HTTP         |       ✅       |       ✅       |
+  | HTTPS        |       ✅       |       ❌       |
+  | WS           |       ✅       |       ✅       |
+  
+  | Routes                                  | Tendermint-Go | Tendermint-Rs |
+  |-----------------------------------------|:-------------:|:-------------:|
+  | [Subscribe](#subscribe)                 |       ✅       |       ✅       |
+  | [Unsubscribe](#unsubscribe)             |       ✅       |       ✅       |
+  | [UnsubscribeAll](#unsubscribeall)       |       ✅       |       ❌       |
```

Comment:
These links seem to go nowhere - did you plan on having a link for them

Reply:
removed these